composer.json

```diff
@@ -10,6 +10,7 @@
         }
     ],
     "require": {
+        "alleyinteractive/wp-asset-manager": "dev-composer",
```

Comment:
Need to update after merging in PR from https://github.com/alleyinteractive/wp-asset-manager